Formula/age-plugin-yubikey.rb

```diff
@@ -0,0 +1,18 @@
+class AgePluginYubikey < Formula
+  desc "Plugin for encrypting files with age and PIV tokens such as YubiKeys"
+  homepage "https://github.com/str4d/age-plugin-yubikey"
+  url "https://github.com/str4d/age-plugin-yubikey/archive/v0.3.0.tar.gz"
+  sha256 "3dfd7923dcbd7b02d0bce1135ff4ba55a7860d8986d1b3b2d113d9553f439ba9"
+  license "MIT"
+  head "https://github.com/str4d/age-plugin-yubikey.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/age-plugin-yubikey --version")
```

Comment:
Any other functionality of the binary will behaved differently depending on whether there is a YubiKey plugged in and on its state. Similarly to `ykman` and `ykpers`, falling back to `--version` for the test.

Reply:
I think we can assume that most of the time this won't be the case. Just add a comment to this extend and expect no yubikey to be present for the test.

Reply:
Added an expect script for the first line of the interactive prompt.